DuckDB.NET.Data/DuckDBAppender.cs

```
@@ -0,0 +1,181 @@
+﻿using System;
+#nullable enable
+namespace DuckDB.NET.Data;
+
+public class DuckDBAppender : IDisposable
+{
+	private readonly DuckDBConnection _connection;
+	private readonly string _table;
+	private readonly NET.DuckDBAppender _nativeAppender;
+
+	internal DuckDBAppender(DuckDBConnection connection, string table)
+	{
+		_connection = connection;
+		_table = table;
+
+		NativeMethods.Appender.DuckDBAppenderCreate(connection.NativeConnection, null, table, out _nativeAppender);
```

You should throw an exception if the return value is an error.